From 77bf3148e3395c4156e48c7b7f363f3e8fa2a03a Mon Sep 17 00:00:00 2001
From: feeRnt <81442162+feeRnt@users.noreply.github.com>
Date: Wed, 8 Apr 2026 19:54:21 +0000
Subject: [PATCH] Licensing: Fix and add licensing information for proprietary
 firmware, and mt76x8 driver
MIME-Version: 1.0
Content-Type: multipart/mixed; boundary="------------false"

This is a multi-part message in MIME format.
--------------false
Content-Type: text/plain; charset=UTF-8; format=fixed
Content-Transfer-Encoding: 8bit


Signed-off-by: feeRnt <81442162+feeRnt@users.noreply.github.com>
---
 .../net/wireless/mediatek/mt76x8/README.license  | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)
 create mode 100644 drivers/net/wireless/mediatek/mt76x8/README.license


--------------false
Content-Type: text/x-patch; name="0065-Licensing-Fix-and-add-licensing-information-for-prop.patch"
Content-Transfer-Encoding: 8bit
Content-Disposition: attachment; filename="0065-Licensing-Fix-and-add-licensing-information-for-prop.patch"

diff --git a/drivers/net/wireless/mediatek/mt76x8/README.license b/drivers/net/wireless/mediatek/mt76x8/README.license
new file mode 100644
index 0000000000000000000000000000000000000000..96c96815ccbc938201a553b31dd5888e94f8326a
--- /dev/null
+++ b/drivers/net/wireless/mediatek/mt76x8/README.license
@@ -0,0 +1,16 @@
+MediaTek driver licensing
+=========================
+
+The source code in this directory and subdirectories
+is provided under a dual license:
+
+  - GNU General Public License version 2 (GPL-2.0)
+  - BSD 3-Clause License
+
+You may choose either license when using or redistributing this code.
+
+When this code is built as part of the Linux kernel, it is
+effectively used under the terms of GPL-2.0.
+
+See the original license header in the source files for the
+full license text.

--------------false--


